Unholster/react-use-config
# Feature/adapter-function

A function to modify the received config is needed. 

Additionally typescript and prettier was added to the project.

# How to test

Build with `build` command.

Install on a project.

Based on the readme, define an adapter function, add a property and verify it was added.

Additionally verify the `lint` commnad.
